api/src/pcapi/core/bookings/api.py

```diff
@@ -358,6 +358,11 @@ def _cancel_booking(
     """Cancel booking and update a user's credit information on Batch"""
     with transaction():
         stock = offers_repository.get_and_lock_stock(stock_id=booking.stockId)
+        # Anytime we call stock we get the last version in the database so we can update the stock quantities correctly
```

Comment:
Je pense qu'on peut faire plus court
```
# Since stock is refreshed every time we access it ([cf ](cf: comment in `offers_repository.get_and_lock_stock`))
# we must fetch the offer separately to avoid duplicating db calls